docs/05-administrator-guide/03-operations/03-node-decommissioning-and-maintenance/01-ozone-manager.md

```diff
@@ -4,4 +4,19 @@ sidebar_label: Ozone Manager
 
 # Decommissioning an Ozone Manager
 
-**TODO:** File a subtask under [HDDS-9859](https://issues.apache.org/jira/browse/HDDS-9859) and complete this page or section.
+Ozone Manager (OM) decommissioning is the process in which you gracefully remove one of the OM from the OM HA Ring.
+
+To decommission an OM and remove the node from the OM HA ring, the following steps need to be executed.
+
```

Comment:
Similar to the SCM Decommission page, recommend transfer the leadership to another OM. (Link to the "Ozone Manager Leader Transfer" page)